src/services/searchArticles.js

```diff
@@ -4,5 +4,6 @@ const API_KEY = process.env.API_KEY;
 
```

Comment:
you could move this to the same file as your other fetch call. Completely up to you but it would  cut down on a file. Again, your choice.